pkgs/development/haskell-modules/configuration-ghc-9.4.x.nix

```diff
@@ -82,21 +81,10 @@ in {
       (dontCheck super.cabal2nix);
   cabal2nix-unstable = dontCheck super.cabal2nix-unstable;
 
-  # build fails on due to ghc api changes
-  # unfinished PR that doesn't yet compile:
-  # https://github.com/sol/doctest/pull/375
-  doctest = markBroken super.doctest_0_20_0;
-  # consequences of doctest breakage follow:
-  http-types = doctest_0_20_broken super.http-types;
-  iproute = doctest_0_20_broken super.iproute;
-  foldl = doctest_0_20_broken super.foldl;
-  prettyprinter-ansi-terminal = doctest_0_20_broken super.prettyprinter-ansi-terminal;
-  pretty-simple = doctest_0_20_broken super.pretty-simple;
-  http-date = doctest_0_20_broken super.http-date;
-  network-byte-order = doctest_0_20_broken super.network-byte-order;
-  co-log-core = doctest_0_20_broken (doJailbreak super.co-log-core);
-  xml-conduit = doctest_0_20_broken (dontCheck super.xml-conduit);
-  validation-selective = doctest_0_20_broken (dontCheck super.validation-selective);
+  doctest = super.doctest;
+  co-log-core = doJailbreak super.co-log-core;
+  xml-conduit = dontCheck super.xml-conduit;
+  validation-selective = dontCheck super.validation-selective;
```

Comment:
These `dontCheck`s were added to avoid depending on doctest, now that it is unbroken we should reenable tests and check if they can now be executed properly.